data.sql

```diff
@@ -47,4 +47,39 @@
     update animals set owner_id = owners.id from owners
     where animals.name in ('Angemon','Boarmon') and owners.full_name = 'Dean Winchester';
 
+    
+vet_clinic=# insert into vets(name,age,date_of_graduation) values
+ ('William Tatcher', 45, '2000-04-23'),
+ ('Maisy Smith', 26 , '2019-01-17'),
+ ('Stephanie Mendez', 64, '1981-05-04'),
+ ('Jack Harkness', 38 , '2008-06-08');
+
+ insert into specializations (vet_name, species_name) values
+ ('William Tatcher','Pokemon'),
```

Comment:
- [ ] After Making the changes to the `specializations` table, you will need to update these queries to insert the correct data into the table, you can use their ids to add the data or you can use their names to get the ids and insert that into the table I.E: 
```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));
```